filebeat/docs/modules/mariadb.asciidoc

```diff
@@ -0,0 +1,75 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-mariadb]]
+:modulename: mariadb
+
+== MariaDB module
+
+The +{modulename}+ module collects and parses the slow logs and error logs
```

Comment:
Should we add a note here on why it's experimental, meaning that we want to merge it into mysql if possible?

Reply:
Yes, I'll add a note about this if this gets merged as a separated module.